tests/testthat/test-list_animal_ids_api.R

```diff
@@ -0,0 +1,17 @@
+test_that("list_animal_ids_api() returns unique list of values", {
```

Comment:
I would not create a new test file for this, but include it in `test-list_animal_ids.R`.